### DISCUSSION TITLE
Schedule documents and blog posts

### DISCUSSION THREAD

**Discussion by andrewpbray** [quarto-dev/quarto-cli#10126] [Show and Tell]
### Description

I've written an extension to Quarto projects that allows you to schedule the rendering of your website or blog. Please check it out!

[https://github.com/qmd-lab/scheduled-docs](https://github.com/qmd-lab/scheduled-docs)

I would like to submit it to be added to the [official list of Quarto Extensions](https://quarto.org/docs/extensions/listing-formats.html), but I'm not sure which category this falls under? It adds project metadata: `pre-render`, `post-render`, and `metadata-files`.

**Comment by mcanouil** [comment]
Well, the list on Quarto is not an "official" list. It's really just a list as any other.
It's not like CRAN, PyPi, etc.

There are no real guidelines or review there, so submit it where you think it is more appropriate.

Same applies with the list I maintain: https://m.canouil.dev/quarto-extensions/
